dev/.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# snapshots
+__snapshots__
```

Comment:
The comment above is not helpful/necessary.

If you need a comment, it should explain why we shouldn't have the `__snapshots__` folder in the git repo. You probably should explain this, because of the other recommendation was to actually put snapshots in the repo; see my previous comment.

Also, remember to use newline at the end of files.